### DISCUSSION TITLE
Plot notifications are missing, I cannot destroy or switch in my own town.

### DISCUSSION THREAD

**Discussion by maunderingcabal** [TownyAdvanced/Towny#5158] [Q&A] [answered]
I use to have this plugin on a server a long time ago and decided to make a new one. Going between town and wilderness no longer displays text warning. Anyone that is not an OP can not build destroy or switch while in their town. Even though the perms are set on... Why does this plugin no longer work? Help please!

**Comment by LlmDl** [comment, marked as answer]
The missing text notification is likely because you're on 1.17 and not using the [pre-release made for 1.17.](https://github.com/TownyAdvanced/Towny/releases) 

No one has had any trouble with destroy/switch in their own towns. Might be another plugin causing that.
